capi/include/blazesym.h

```diff
@@ -150,6 +150,16 @@ typedef struct blaze_normalizer_opts {
    * ensure compatibility in the presence of member additions.
    */
   size_t type_size;
+  /**
+   * Whether or not to always re-read and re-parse `/proc/<pid>/maps` for
+   * each request.
+   *
+   * Setting this flag to `false` will cause the parsed entries to be cached.
+   * This is not generally recommended, because it could mean that addresses
+   * corresponding to recently added mappings cannot be normalized
+   * successfully (as there is no reasonable way of detecting staleness).
+   */
+  bool reevaluate_maps;
```

Comment:
As a user, I find more intuitive the opposite logic and call this `cache_maps`, but I understand this is subjective.

Reply:
Yes, but we generally have positive logic that is enabled by default and I wanted to keep it that way. Not particularly happy with the name either, though.

Reply:
I changed it after conversation with @anakryiko